cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,39 @@
 describe('', () => {
-  before(() => {
-
+  let user;
+  let article;
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
   });
 
-  it('', () => {
-
+  it('allow to create the article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.visit('/editor');
+    cy.findByPlaceholder('Article Title').type(article.title);
+    cy.findByPlaceholder('What\'s this article about?')
+      .type(article.description);
+    cy.findByPlaceholder('Write your article (in markdown)').type(article.body);
+    cy.get('.btn').contains('Publish Article').click();
+    cy.get('.container').should('contain', article.title);
+    cy.get('.author').should('contain', user.username.toLowerCase());
   });
 
-  it('', () => {
-
+  it('allow to delete the article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.createArticle(article.title, article.description, article.body);
```

Comment:
```suggestion
    cy.createArticle(article.title, article.description, article.body);
```
try to extract article slug from that  response and visit article page directly